The job **`x86_64-gnu-tools`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/12384834039/job/34570066948) [(plain)](https://github.com/rust-lang/rust/commit/2903356b2e0d73089d66e24b61bfd47d22467851/checks/34570066948/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
Verifying status of embedded-book...
Cloning into 'rust-toolstate'...
error: RPC failed; HTTP 503 curl 22 The requested URL returned error: 503
fatal: expected flush after ref listing
Command has failed. Rerun with -v to see more details.
  local time: Wed Dec 18 02:13:43 UTC 2024
  network time: Wed, 18 Dec 2024 02:13:43 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/compiler/rustc_mir_dataflow/src/framework/visitor.rs
+++ b/compiler/rustc_mir_dataflow/src/framework/visitor.rs
@@ -35,7 +35,6 @@ where
 {
     fn visit_block_start(&mut self, _state: &A::Domain) {}
 
-    /// // njn: grep for "before", "primary", etc.
     /// Called after the "early" effect of the given statement is applied to `state`.
     fn visit_after_early_statement_effect(
         &mut self,

